### PR TITLE
[#76] [UI] Create smiley rating bar (Part 4)

### PR DESCRIPTION
**Pull request by sleepylee**
Part of #76 

## What happened 👀

Create a smiley choser/rating bar.
 
## Insight 📝

- Single selection smiley on a pre-defined scale (counter).
- Add emoji and state disabled/enabled accordingly.
 
## Proof Of Work 📹

https://user-images.githubusercontent.com/13435717/125051585-4a09b600-e0cd-11eb-8a61-d1f554950acf.mp4

### CODE REVIEW

File: lib/pages/ui/smiley_rating_bar.dart [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      case 1:
+        return Text("😕", style: correctStyle);
+      case 2:
+        return Text("😐", style: correctStyle);
+      case 3:
+        return Text("🙂", style: correctStyle);
+      case 4:
+      default:
+        return Text("😄", style: correctStyle);
+    }
```
Comment (markgravity) [inline lib/pages/ui/smiley_rating_bar.dart:71]: We can save some lines of code by adding these emojis into an array. What do you think?
```dart
const emojis = ["😡","😕" ,"😐","🙂","😄"]
return Text(emojis[index], style: correctStyle);
```

**Review by markgravity — APPROVED** [review]
LGTM 🤖

File: lib/pages/ui/smiley_rating_bar.dart (hunk 1 above)
Comment (sleepylee) [reply lib/pages/ui/smiley_rating_bar.dart:71]: ok I think maybe Map could be better 😂  at least we know what is the meaning behind it I guess, let me update it

**Comment by sleepylee** [comment]
all updated 🙇  including the state accessibility refactor

**Review by manh-t — APPROVED** [review]

**Pull request merged**